src/app.js

```diff
@@ -0,0 +1,23 @@
+import {
+  // eslint-disable-next-line import/named
```

Comment:
- Good job working on this milestone 💪🏿 

- [ ]  It would be better to not disable linters manually, please ensure that your linters pass the green checks ✅